_data/versions.yml

```diff
@@ -3,30 +3,22 @@ v3.5:
   components:
     calico/node:
       version: v3.5.0
-      url: https://github.com/projectcalico/node/releases/tag/v3.5.0
     calicoctl:
       version: v3.5.0
-      url: https://github.com/projectcalico/calicoctl/releases/tag/v3.5.0
     calico/cni:
       version: v3.5.0
-      url: https://github.com/projectcalico/cni-plugin/releases/tag/v3.5.0
     calico/kube-controllers:
       version: v3.5.0
-      url: https://github.com/projectcalico/k8s-policy/releases/tag/v3.5.0
     networking-calico:
       version: v3.4.0
```

Comment:
We should probably drop the `v` from networking-calico - I think that's just a bug.